### PR TITLE
Fix benchmark suite after emmake changes

### PR DESCRIPTION
**Pull request by kripken**
Recent test runner changes require us to make EMMAKE explicitly.

### CODE REVIEW

**Review by sbc100 — APPROVED** [review]
Thanks!

(I do think its clearer to do it this way, sorry for the breakage)

File: tests/test_benchmark.py [hunk 1]
```diff
@@ -1022,7 +1040,13 @@ def test_zzz_lzma(self):
     src = read_file(test_file('benchmark', 'test_lzma_benchmark.c'))
 
     def lib_builder(name, native, env_init):
-      return self.get_library(os.path.join('third_party', 'lzma'), [os.path.join('lzma.a')], configure=None, native=native, cache_name_extra=name, env_init=env_init)
+      return self.get_library(os.path.join('third_party', 'lzma'),
```
Comment (sbc100) [inline tests/test_benchmark.py:1043]: I think we might want to do something like `make = ['make'] if native else [EMMAKE, 'make']` so continue to support the native build.   We could perhaps find a way to make that the default.

Comment (sbc100) [reply tests/test_benchmark.py:1043]: Actually I think I have better fix..

**Comment by kripken** [comment]
The other PR is indeed better...

**Pull request closed without merging**